MapboxNavigation/Abbreviations.swift

```diff
@@ -34,17 +34,19 @@ extension String {
     }
     
     /// Returns the string abbreviated only as much as necessary to fit the given width and font.
-    func abbreviated(width: CGFloat, font: UIFont) -> String {
+    func abbreviated(bounds: CGRect, font: UIFont) -> String {
```

Comment:
This method should be called `abbreviated(toFit:font:)`, not `abbreviated(bounds:font:)`. The relationship between the method name and the argument value should be clear when reading the method call.